source/_integrations/image_processing.markdown

```diff
@@ -5,6 +5,7 @@ ha_category:
   - Image Processing
 ha_release: 0.36
 ha_domain: image_processing
+ha_iot_class: Calculated
```

Comment:
This integration provides a base for other integration to build on. It cannot be used stand-alone, and therefore has no IoT class.

```suggestion
ha_iot_class: ~
```